gui/src/components/onboarding/pages/stay-aligned/stay-aligned-steps/VerifyMounting.tsx

```diff
@@ -1,32 +1,37 @@
+import { useState } from 'react';
 import { Button } from '@/components/commons/Button';
 import { Typography } from '@/components/commons/Typography';
 import { ResetType } from 'solarxr-protocol';
 import { ResetButton } from '@/components/home/ResetButton';
-import { useLocalization } from '@fluent/react';
 import { useBreakpoint } from '@/hooks/breakpoint';
 import { VerticalStepComponentProps } from '@/components/commons/VerticalStepper';
-
+import { BaseModal } from '@/components/commons/BaseModal';
+import { ManualMountingPageStayAlligned } from '@/components/onboarding/pages/mounting/ManualMounting';
 export function VerifyMountingStep({
   nextStep,
   prevStep,
 }: VerticalStepComponentProps) {
   const { isMobile } = useBreakpoint('mobile');
-  const { l10n } = useLocalization();
+  const [isOpen, setOpen] = useState(false);
+
+  function OpenFunc() {
+    setOpen(true);
+  }
+  function ProgressFunc() {
+    setOpen(false);
+    nextStep();
+  }
+  function CloseFunc() {
+    setOpen(false);
+  }
+
```

Comment:
i don't think we need wrappers around a one-liner, we can just inline this

```suggestion
  const goNextStep = () => {
    setOpen(false);
    nextStep();
  };

```